src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/Head.java

```diff
@@ -328,9 +328,7 @@ private void addStylesheets(HtmlTree head) {
         }
 
         if (index) {
-            // The order of the addStylesheet(...) calls is important
             addStylesheet(head, DocPaths.SCRIPT_DIR.resolve(DocPaths.JQUERY_UI_CSS));
-            addStylesheet(head, DocPaths.JQUERY_OVERRIDES_CSS);
```

Comment:
The edit as-is is OK, but the general code here makes me wonder if this class belongs in the `markup` package.

Reply:
I don't see the problem. The `Head` class generates part of the page markup, so the package seems fitting to me.

Reply:
It's in a gray area. The `markup` package was supposed to be low-level primitives for building basic HTML trees ... and these lines reflect policy more than underlying support. We moved `Table` and `Navigation` out of `markup` for the same reason.   Anyway, this is just a discussion, not a blocker.